pornhub/stars.py

```diff
@@ -7,18 +7,43 @@ class Stars(object):
     def __init__(self, ProxyDictionary, *args):
         self.ProxyDictionary = ProxyDictionary
 
-    def _loadStarsPage(self, page_num):
-        payload = { "page" : page_num }
-        r = requests.get(BASE_URL + PORNSTARS_URL, params=payload, headers=HEADERS, proxies=self.ProxyDictionary)
-        html = r.text
+    def _sortStars(self, sort_by):
+        sort_dict = dict()
+
+        if not sort_by:
+            return sort_dict
+        
+        sort_types = {"view": "mv", "trend": "t", "subs": "ms", "alpha": "a", "videos": "nv", "random": "r"}
 
+        for key in sort_types:
+            if key in sort_by.lower():
+                sort_dict["o"] = sort_types[key]
```

Comment:
I don't quite understand this piece of code. Why are you always updating the same key in `sort_dict`?

Reply:
Maybe there isn't a need to loop through the whole dictionary in the first place?

Reply:
I did that in a such way for that in `sort_by` it was possible to specify the type of sorting not strictly, for example, you can write `sort_by = "views"`or `sort_by = "viewed"` and it will still work. Maybe there is a more elegant way to write this, but I don't know it. с:

Reply:
Okay, it's fine for now.